### DISCUSSION TITLE
Smoke shakes when gameplay is paused

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20686] [Q&A] [answered]
As title. Also, if an overlay temporarily shifts the playfield, the smoke stops wobbling for that moment:

![osu! 2022 10 10 - 12 06 45 01_Trim](https://user-images.githubusercontent.com/43073074/194843639-06db8027-5b8b-49da-a54c-9e8a7bb5d682.gif)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20705
